docs/building.md

```diff
@@ -173,22 +181,30 @@ cd devilutionx
 
 ### Installing dependencies on WSL, Debian and Ubuntu
 
+The following compilation process has been tested on `ubuntu-22.04`. Success is not guaranteed on Linux distributions other than Ubuntu 22.
```

Comment:
```suggestion
The following compilation process has been tested on `ubuntu-22.04`. Success is not guaranteed on Linux distributions other than Ubuntu 22.04
```

Reply:
@ZhiZe-ZG Ubuntu's version are YY.MM, there is one every 6 month, and a long term every 2 years.

23.10 = October 2023
24.04 LTS = April 2024
24.10 = October 2024
...
26.04 LTS = April 2026

So that's why the minor version (month) is important to include and not just the major (year).

Reply:
Thanks